standalone/include/psen_scan_v2_standalone/data_conversion_layer/io_pin_data.h

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#ifndef PSEN_SCAN_V2_STANDALONE_IO_PIN_H
+#define PSEN_SCAN_V2_STANDALONE_IO_PIN_H
+
+#include <functional>
+#include <stdexcept>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "psen_scan_v2_standalone/data_conversion_layer/io_constants.h"
+#include "psen_scan_v2_standalone/io_state.h"
+
+namespace psen_scan_v2_standalone
+{
+namespace data_conversion_layer
+{
+namespace monitoring_frame
+{
+/**
+ * @brief Contains all types, etc. needed to describe the IOs information contained
+ * in a  data_conversion_layer::monitoring_frame::Message.
+ */
+namespace io
+{
+static uint32_t createID(size_t byte_n, size_t bit_n)
+{
+  return byte_n * 8 + bit_n;
```

Comment:
Why is this id needed? What is the use case?

Reply:
Maybe if this is provided somethink like https://answers.ros.org/question/9427/enum-in-msg/ would be useful for ROS Users and keep the code stable

Reply:
@jschleicher what do you think?

Reply:
I'd suggest to move the list from  [Readme ](https://github.com/PilzDE/psen_scan_v2/blob/7a7e77652c067d4ebc85d414c63257dd119240b1/README.md#transferred-ios) into the .msg message definition. So that the user can deal with constants in a similar way as in e.g. [MoveitError-Codes](https://github.com/ros-planning/moveit_msgs/blob/master/msg/MoveItErrorCodes.msg).